README.md

```diff
@@ -61,6 +61,19 @@ For more info, see:
 - [@nx/node:setup-docker](https://nx.dev/nx-api/node/generators/library): Generator for NodeJS Libraries
 - [@nx/node:setup-docker](https://nx.dev/nx-api/node/generators/setup-docker): Docker Generator
 
+## Test
+
+ ```
+ # Run all tests
+ yarn test
+
+ # Run test for repositories
+ nx run repositories:test --watch
+
+ # Run test on specific file
+ nx run repositories:test --watch --testFile=libs/repositories/src/UsdRepository/UsdRepositoryRedis.spec.ts
```

Comment:
Why default the test with `--watch`?

Reply:
I mean, I didn't add it as default, I just gave 3 representative examples. One runs all tests and finishes, another runs the test in watch mode for repository layer, last does the same for a specific file. 

I consider this notes just hints for devs on somehing I found useful for myself

I will add some additional text in the comment to clarify